helmfile/charts/networkpolicy-service/templates/rook-ceph/mgr.yaml

```diff
@@ -0,0 +1,119 @@
+{{ if .Values.rookCeph.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-rook-ceph-mgr
+  namespace: rook-ceph
+spec:
+  policyTypes:
+    - Ingress
+    - Egress
+  podSelector:
+    matchLabels:
+      app: rook-ceph-mgr
+  ingress:
+    - from:
```

Comment:
You can group all `from` using the same `ports` together as one block instead of separate has you have now.

Reply:
@robinAwallace thanks !! I have done the changes now , have a look